src/model/pad_options/options.go

```diff
@@ -9,81 +9,112 @@ import (
 	"os"
        _ "github.com/go-sql-driver/mysql"
 	"database/sql"
+	"github.com/lucasjones/reggen"
+	"strconv"
 
 )
+type Pad struct{
+	ID string `json:"id"`
+	Name string `json:"name"`
+	Value string `json:"value"`
+}
+
+
+type Optionss_Fun interface {
+	StorePad(w http.ResponseWriter ,r *http.Request, _ httprouter.Params)
+	RenameFile(w http.ResponseWriter ,r *http.Request, _ httprouter.Params)
+	DeleteFile(w http.ResponseWriter ,r *http.Request, _ httprouter.Params)
+	EmptyDocument(w http.ResponseWriter ,r *http.Request, _ httprouter.Params)
 
+}
 var (
 	fileInfo *os.FileInfo
 	err      error
 )
 
 
+func NewPad() *Pad{
+	return  &Pad{}
+}
+
 type PadRequest struct{
```

Comment:
possible changes in this struct??? request won't have only the ID pad bad info regarding IP address of client requesting
(needed to update history)